sechub-server-core/build.gradle

```diff
@@ -3,8 +3,8 @@ dependencies {
 
     api(library.springboot_starter_batch)
     
-    implementation 'org.springframework.boot:spring-boot-starter-actuator'
-    implementation 'io.micrometer:micrometer-registry-prometheus'
+	implementation library.springboot_starter_actuator
```

Comment:
Please have a look if only github displays this incorrectly or if there is a problem with indentation.